src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -118,9 +118,13 @@ def validation_error(self, message):
 
     def error(self, message):
         telemetry.set_user_fault('parse error: {}'.format(message))
-        args = {'prog': self.prog, 'message': message}
-        logger.error('%(prog)s: error: %(message)s', args)
-        self.print_usage(sys.stderr)
+        if message.endswith('_subcommand'):
+            error_msg = "{prog}: Missing a subcommand. See '{prog} --help'.".format(prog=self.prog)
```

Comment:
Why would we not print usage? Now we *force* them to re-run with -h, when they could otherwise figure out what they want from the usage string.